### PR TITLE
Create "marketplace" for console integrations

### PR DESCRIPTION
**Pull request by vicmgs**
~~test by copying templates/allowed-integrations.json into /config folder and tweaking it to select allowed types~~

### CODE REVIEW

**Review by ambuznego — APPROVED** [review]

File: assets/js/components/channels/ChannelNewCommunityIntro.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+              overflow: "hidden",
+              boxShadow: "0px 20px 20px -7px rgba(17, 24, 31, 0.19)",
+            }}
+          >
+            <div style={{ overflowX: "scroll" }} className="no-scroll-bar">
+              <div style={{ padding: "60px 30px 60px 30px", minWidth }}>
+                <Row>
+                  <Col span={12}>
+                    <Text style={{ fontSize: 22, fontWeight: 600, display: 'block' }}>{integrationType.info.title}</Text>
+                    <Text style={{ display: 'block', marginTop: 30 }}>{integrationType.info.desc}</Text>
```
Comment (ambuznego) [inline assets/js/components/channels/ChannelNewCommunityIntro.jsx]: make this text 16px to match designs

File: assets/js/components/channels/ChannelNewCommunityIntro.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+              borderRadius: 6,
+              overflow: "hidden",
+              boxShadow: "0px 20px 20px -7px rgba(17, 24, 31, 0.19)",
+            }}
+          >
+            <div style={{ overflowX: "scroll" }} className="no-scroll-bar">
+              <div style={{ padding: "60px 30px 60px 30px", minWidth }}>
+                <Row>
+                  <Col span={12}>
+                    <Text style={{ fontSize: 22, fontWeight: 600, display: 'block' }}>{integrationType.info.title}</Text>
```
Comment (ambuznego) [inline assets/js/components/channels/ChannelNewCommunityIntro.jsx]: 28px font size to match designs as well

File: assets/js/components/channels/ChannelNewCommunityIntro.jsx [hunk 3]
```diff
@@ -20,7 +20,54 @@ const ChannelNewCommunityIntro = (props) => {
     <>
       <MobileDisplay>
         <MobileLayout>
-          <div>agsf</div>
+          <div
+            style={{
+              padding: "10px 15px",
```
Comment (ambuznego) [inline assets/js/components/channels/ChannelNewCommunityIntro.jsx:25]: might need a bit more padding here:
image

File: assets/js/components/channels/ChannelNewCommunityIntro.jsx [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+      </DesktopDisplay>
+    </>
+  )
+}
+
+const renderRow = (integrationType, history, mobile = false) => (
+  <Row>
+    <Col span={mobile ? 24 : 12}>
+      <Text style={{ fontSize: 22, fontWeight: 600, display: 'block' }}>{integrationType.info.title}</Text>
+      <Text style={{ display: 'block', marginTop: 30 }}>{integrationType.info.desc}</Text>
```
Comment (ambuznego) [inline assets/js/components/channels/ChannelNewCommunityIntro.jsx:124]: to match designs: 
title -> 28px
desc -> 16px

**Comment by ambuznego** [comment]
for consistency with this section, could we move the "turned off" warning to the warnings section: 
image

File: assets/js/components/channels/ChannelNewCommunityIntro.jsx (hunk 3 above)
Comment (vicmgs) [reply assets/js/components/channels/ChannelNewCommunityIntro.jsx:25]: this padding is exactly the same as on other pages

File: assets/js/components/channels/ChannelNewCommunityIntro.jsx (hunk 4 above)
Comment (vicmgs) [reply assets/js/components/channels/ChannelNewCommunityIntro.jsx:124]: made it smaller on mobile since 28 looks really ugly

**Pull request merged**